<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShrewDriver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Air Puff:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Animal Name: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="cbAnimalName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Task Type:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="cbTaskType"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Sensors:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="cbSensors"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Syringe Pump (L):</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="cbLeftSyringePump"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Syringe Pump (R):</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="cbRightSyringePump">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Visual Stim:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="cbVisualStim"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Eye tracking Cam:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Observation Cam:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QComboBox" name="cbAirPuff"/>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QPushButton" name="btnSelectROI">
      <property name="text">
       <string>Select ROI</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="3">
     <widget class="QTextBrowser" name="txtTrialStats">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="3">
     <widget class="QPushButton" name="btnStartRecording">
      <property name="text">
       <string>Start Recording</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="cbEyeCameraID"/>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="chkbOpto">
      <property name="text">
       <string>Use opto?</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QComboBox" name="cbBehaviorCameraID"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Opto Experiment:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="chkbOptoFS">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Only first stim?</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>89</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>89</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Trial Stats:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QPushButton" name="btnNewCalibration">
      <property name="text">
       <string>New Calibration</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>296</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbEyeCameraID</tabstop>
  <tabstop>cbAnimalName</tabstop>
  <tabstop>cbSensors</tabstop>
  <tabstop>cbVisualStim</tabstop>
  <tabstop>cbLeftSyringePump</tabstop>
  <tabstop>cbAirPuff</tabstop>
  <tabstop>cbRightSyringePump</tabstop>
  <tabstop>txtTrialStats</tabstop>
  <tabstop>btnStartRecording</tabstop>
  <tabstop>cbTaskType</tabstop>
  <tabstop>btnSelectROI</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
